_data/whatis.yml

```diff
@@ -0,0 +1,7 @@
+content: <p>The podcast that takes a deeper look into all things tech with
+  those who make it happen. Weekly we talk about the latest
+  developments in technology around the world, and we have a
+  fantastic guest talking about the part they play in the tech
+  industry and how it's shaped their lives.</p>
+  <br/>
```

Comment:
Not a correct use of `br` We should use CSS on p tags to make this a universal style for paragraphs following paragraphs

Reply:
fair enough, can change that. I haven't really written HTML in a while...